defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+nmcli_add_addrs_interface: "{{ omit }}"
```

Comment:
I believe, in other places we've run into this type of usage, we've defaulted to nothing here and used the `| default(omit)` filter on the task itself. Still have the variable defined, but just comment it out as such:

`# nmcli_add_addrs_interface: eth0`

@robled and @andrew103 have encountered this before and came up with an existing paradigm. So long as this one matches that, we're all good (and should document it in meta_standards).

Reply:
I like that style, and agree that we should document it. In this case, though, it leads to ugly logic in the tasks (`(var | default(omit) != omit`). I've added comments (woops) to `defaults/main.yml` to explain why they default to omit, and also explain in a little more detail how the interface and connection are related to each other, and why at least one must be set.

Reply:
So in the `when` clause and the assert `that` clause, you can directly test "var is defined" as one would do in pseudocode somewhere (analogous to `is not None` in Python or `typeof foo !== 'undefined'` in JavaScript). That allows you to directly put `{{ var | default(omit) }}` into places where it's an optional argument for a parameter and simply leave it undefined in `defaults/main.yml` for places where on sane default exists. You can then test for `is defined` and win at all the things.